decorators.py

```diff
@@ -38,3 +38,12 @@ def representative_or_staff(function=None, redirect_field_name=REDIRECT_FIELD_NA
     if function:
         return actual_decorator(function)
     return actual_decorator
+
+def is_volunteer(function=None, redirect_field_name=REDIRECT_FIELD_NAME):
```

Comment:
We can call this method `volunteer_required` so that it follows the same format as the other decorators.

Also it will throw an exception if the user doesn't have a profile (as is the case with some admins) - can you catch that and return `False` instead?